3-networks/README.md

```diff
@@ -22,9 +22,11 @@ The purpose of this step is to :
 1. Copy cloud build configuration files for terraform `cp ../terraform-example-foundation/build/cloudbuild-tf-* . ` (modify accordingly based on your current directory)
 1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` (modify accordingly based on your current directory)
 1. Ensure wrapper script can be executed `chmod 755 ./tf-wrapper.sh`.
-1. Within each envs/ folder rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap.
+1. Rename `common.auto.example.tfvars` to `common.auto.tfvars` and update the file with values from your environment and bootstrap.
+1. Rename `dev_nonprod_prod.auto.example.tfvars` to `dev_nonprod_prod.auto.tfvars` and update the file with the `access_context_manager_policy_id` value.
```

Comment:
These steps need clarification, because the files are not called like that - Amanda is working on a fix - https://github.com/terraform-google-modules/terraform-example-foundation/issues/169#issuecomment-667076579

Reply:
@amandakarina and @atos-cit - FYI

Reply:
Got the most recent changes from Amanda already. Thank you.